src/org.xtuml.bp.core.test/generate.xml

```diff
@@ -8,9 +8,15 @@
   <target name="all" description="Generate all code for this project" depends="dispose_test,action_test,delete_test,formalize_test,context_menu_tests,copy_paste_tests, rto_move_tests, consistency_all">
   </target>
 
+  <target name="nb_all" description="Generate all code for this project" depends="dispose_test,action_test,delete_test,formalize_test,context_menu_tests,copy_paste_tests, rto_move_tests, nb_consistency_all">
```

Comment:
I removed this target from bp.core/generate.xml.  Was I incorrect to do that?  What is this target used for here?